modules/core/output_modules.php

```diff
@@ -164,7 +164,28 @@ class Hm_Output_login_end extends Hm_Output_Module {
      * Closes the login form
      */
     protected function output() {
-        return '</form>';
+        $js = '<script type="text/javascript">
```

Comment:
I don't get well this. What is the purpose ? Also we have Jquery in the project why not use it here ?